From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 4 Sep 2013 08:59:37 +0200
Subject: [PATCH] zynq: Increase number of arguments

The main reason is that with adding option to load
variables from SD user can create longer user commands.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Upstream-Status: Backport
---
 include/configs/zynq_common.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/zynq_common.h b/include/configs/zynq_common.h
index a7cadc4..e83c6de 100644
--- a/include/configs/zynq_common.h
+++ b/include/configs/zynq_common.h
@@ -105,7 +105,7 @@
 #define CONFIG_AUTO_COMPLETE
 #define CONFIG_SYS_LONGHELP
 #define CONFIG_BOARD_LATE_INIT
-#define CONFIG_SYS_MAXARGS		16
+#define CONFIG_SYS_MAXARGS		32
 #define CONFIG_SYS_CBSIZE		2048
 #define CONFIG_SYS_PBSIZE		(CONFIG_SYS_CBSIZE + \
 					sizeof(CONFIG_SYS_PROMPT) + 16)
-- 
1.7.9.5

